portal-web/test/functional/com/liferay/portalweb/functions/MouseOver.function

```diff
@@ -50,6 +50,20 @@ definition {
 		selenium.assertLiferayErrors();
 	}
 
+	function mouseUp {
```

Comment:
That's odd that no mouseover function works for this bar graph. Would this do the trick instead: https://github.com/liferay/liferay-portal/blob/master/portal-web/test/functional/com/liferay/portalweb/functions/MouseUp.function#L4

Looks similar to what you want to implement

Reply:
Good catch, Josh. I will check if this works for us

Reply:
Hey Josh, your suggestion is working I updated the PR. Thank you for that